src/app/appStore.js

```diff
@@ -1,4 +1,4 @@
-import { vuex as cartStore } from './cart';
+//import { vuex as cartStore } from './cart';
 import { vuex as productStore } from './product';
 
-export default { ...cartStore, ...productStore };
```

Comment:
It's no needed to *"copy"* the module, with rest parameter, since we just importing and export right away